gulpfile.js

```diff
@@ -52,7 +52,7 @@ gulp.task('clean-release', clean_release);
 
 gulp.task('clean-cache', clean_cache);
 
-var distBuild = gulp.series(clean_dist, dist);
+var distBuild = gulp.series(clean_dist, dist_src, dist_locale, dist_libraries, dist_resources);
```

Comment:
I don't know if it will be a good idea, but we can execute this tasks in parallel, if I'm not wrong:
>var distBuild = gulp.series(clean_dist, gulp.parallel(dist_src, dist_locale, dist_libraries, dist_resources));

Reply:
These tasks could run parallel to each other, the do not have conflicting actions at this point.
In my opinion there is less benefit, as the whole `dist` task runs in in less than 700ms.